src/Features/Core/Portable/UnusedReferences/UpdateAction.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal enum UpdateAction
+    {
+        /// <summary>
+        /// No action needs to be performed.
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Indicates the reference needs to be updated.
+        /// </summary>
+        Update,
```

Comment:
What's an example of this?

Reply:
One of the goals is to have the ability to tag references as "TreatAsUsed" (or some similar attribute) to indicate that although the reference is unused at compile time, it should be considered used for the purposes of analyzing unused references. This update action would be sent to the project system after a user has changed an unused reference's treated as used state.